lib/http2.js

```diff
@@ -28,4 +30,6 @@ module.exports = {
   connect,
   Http2ServerResponse,
   Http2ServerRequest,
+  Http2Session,
```

Comment:
I'm fine with this - but increasing the surface of the API is not trivial - and I'd like to see docs.